osu.Game/Graphics/Backgrounds/Triangles.cs

```diff
@@ -63,8 +70,11 @@ protected override void Update()
         {
             base.Update();
 
+            float adjustedAlpha = ExponentialAlphaAdjust ? (float)Math.Pow(((Color4)DrawInfo.Colour.Colour).A, 3) : 1;

```

Comment:
I suppose it's not even much of a discrepancy, but the name is way better. Let's move it into `Drawable` and apply it in the `DrawInfo` getter. This is a general thing we might want in other places.